test/unit/featuregeometrylayer.js

```diff
@@ -3,71 +3,85 @@ import assert from 'assert';
 import GlobeView from 'Core/Prefab/GlobeView';
 import FeatureGeometryLayer from 'Layer/FeatureGeometryLayer';
 import FileSource from 'Source/FileSource';
-import { HttpsProxyAgent } from 'https-proxy-agent';
+import { supportedFetchers } from 'Source/Source';
 import Extent from 'Core/Geographic/Extent';
 import Coordinates from 'Core/Geographic/Coordinates';
 import OBB from 'Renderer/OBB';
 import TileMesh from 'Core/TileMesh';
-import Style from 'Core/Style';
+import sinon from 'sinon';
 import Renderer from './bootstrap';
 
+import feature from '../data/filesource/feat_Polygone.geojson';
+
 describe('Layer with Feature process', function () {
     const renderer = new Renderer();
-
     const placement = { coord: new Coordinates('EPSG:4326', 1.5, 43), range: 300000 };
     const viewer = new GlobeView(renderer.domElement, placement, { renderer });
 
-    const source = new FileSource({
-        url: 'https://raw.githubusercontent.com/gregoiredavid/france-geojson/master/departements/09-ariege/departement-09-ariege.geojson',
-        crs: 'EPSG:4326',
-        format: 'application/json',
-        networkOptions: process.env.HTTPS_PROXY ? { agent: new HttpsProxyAgent(process.env.HTTPS_PROXY) } : {},
-    });
+    let ariege;
+    let ariegeNoProj4;
+    let tile;
+    let context;
+    let stubSuppFetcher;
 
-    const source2 = new FileSource({
-        url: 'https://raw.githubusercontent.com/gregoiredavid/france-geojson/master/departements/09-ariege/departement-09-ariege.geojson',
-        crs: 'EPSG:4326',
-        format: 'application/json',
-        networkOptions: process.env.HTTPS_PROXY ? { agent: new HttpsProxyAgent(process.env.HTTPS_PROXY) } : {},
-    });
+    before(function () {
+        stubSuppFetcher = sinon.stub(supportedFetchers, 'get')
+            .callsFake((format) => {
+                if (format === 'application/json') {
```

Comment:
same question, limit call using `withArgs` ? I don't know which is best

Reply:
done (and in all other test files)